.github/workflows/build.yaml

```diff
@@ -6,26 +6,31 @@ on:
     tags:
       - v*
 
+env:
+  MAIN_REGISTRY: ghcr.io
+  MAIN_USERNAME: GHCR_USERNAME
+  MAIN_PASSWORD: GITHUB_TOKEN
```

Comment:
See below, let's make it as descriptive as possible and expand these directly here (no `vars` and `secrets` magic hidden within job specs.

Reply:
Changed in the latest commit.